### PR TITLE
 T840 Remove duplicated hint

### PR DESCRIPTION
**Pull request by OndraZizka**
Not sure why @mareknovotny closed #200. So this is a continuation.

### CODE REVIEW

File: rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```
Comment (mareknovotny) [inline rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml]: the file name with comma is really ugly. Please remove/replace it with something else.

File: rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            The purpose of this ruleset common to those two major JBoss EAP versions is to avoid duplication of rules
+            in rulesets dedicated to either version.
+            Mainly deals with datasources, JBoss Web and EJB configuration.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.6.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.6.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="[5,7)"/>
+        <targetTechnology id="eap" versionRange="[7,)"/>
```
Comment (mareknovotny) [inline rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml]: shouldn't be here also 6? it was for eap5->6 and eap5 -> 7 AFAIK

File: rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml (hunk 1 above)
Comment (OndraZizka) [inline rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml]: Ok, jboss-eap5and6to7-xml.windup.xml

Comment (OndraZizka) [inline rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml]: And needs to change the tests.

**Comment by OndraZizka** [comment]
Let's see what the tests say now

**Comment by OndraZizka** [comment]
Interesting, here the tests passed and locally I got 

  WindupRulesTest.testWindupRules:144 
        Error with test: hsearch-00000-test
        Cause: (Rule: hsearch-00000-test) Assertion failed: hsearch-00000 hint not found!

**Comment by mareknovotny** [comment]
Current master has got 2 failing tests:
```
Failed tests: 
  WindupRulesTest.testWindupRules:144 
	Error with test: websphere-xml-06000-cl-test
	Cause: (Rule: websphere-xml-06000-cl-test) Assertion failed: Classification for WebSphere binding config was not detected!

	Error with test: hsearch-00000-test
	Cause: (Rule: hsearch-00000-test) Assertion failed: hsearch-00000 hint not found!


Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

**Pull request merged**